test/viaq-data-model.sh

```diff
@@ -215,3 +215,12 @@ if [ -n "$is_maximal" ] ; then
     os::cmd::try_until_failure "oc get pod $muxpod"
     os::cmd::try_until_text "oc get pods -l component=mux" "^logging-mux-.* Running "
 fi
+
+# TEST 6
+# catch logs at the debug level
+TEST_REC_PRIORITY=debug wait_for_fluentd_to_catch_up get_logmessage get_logmessage2
```

Comment:
This test is consistently failing.  I think this is probably not a good place to add this test.  The viaq test leaves fluentd and mux in a "bad" state, which is reverted in the cleanup method.

I'm not sure which test you should add this to.  It isn't really related to the viaq data model.  I suppose if you want to add this test to this file, it should be the first test in the file, before the test modifies fluentd and mux, but again, this isn't related.

Reply:
I moved the test to he beginning for now (I made it `TEST 0`) to see if it will pass. Let's find a better place for it before we merge this in...